### PR TITLE
#78 Registering a service into Eureka

### PR DESCRIPTION
**Pull request by Moses-odessa**
connect to #78

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -16,7 +16,7 @@ repositories {
 
 apply plugin: 'java'
 apply plugin: 'idea'
-apply plugin: 'spring-boot'
+apply plugin: 'org.springframework.boot'
```
Comment (adaleorion) [inline build.gradle:18]: Why did you change this line?

Comment (Moses-odessa) [reply build.gradle:18]: BootRun says:
"The plugin id 'spring-boot' is deprecated. Please use 'org.springframework.boot' instead."

Comment (adaleorion) [reply build.gradle:18]: thanks
